CONTRIBUTING.md

```diff
@@ -87,20 +123,20 @@ Alternatively, you can do most of the process using the command line:
 
 - fork the repository on the github web interface
 
-- clone your fork locally:  
+- clone your fork locally:
```

Comment:
Blame my editor for it. :stuck_out_tongue_closed_eyes: Its configured to remove trailing spaces at end of lines. 

Thanks for pointing out the feature.